docs/core/tutorials/netcore-hosting.md

```diff
@@ -45,6 +45,20 @@ The `hostfxr` library is found using `get_hostfxr_path` API from `nethost` libra
 
 [!code-cpp[HostFxrHost#LoadHostFxr](~/samples/snippets/core/tutorials/netcore-hosting/csharp/HostWithHostFxr/src/NativeHost/nativehost.cpp#LoadHostFxr)]
 
+The sample uses the following includes:
+
+```cpp
+#include <nethost.h>
+#include <coreclr_delegates.h>
+#include <hostfxr.h>
+```
+
+These files can be found at the following locations:
+
+* `https://github.com/dotnet/runtime/blob/main/src/native/corehost/nethost/nethost.h`
```

Comment:
Any reason not to make these actual links?

Reply:
Markdownlint complains about naked URLs, so I would have to repeat the URL or not show it, and neither of those options seemed appealing.

Reply:
You can do this:

```suggestion
* <https://github.com/dotnet/runtime/blob/main/src/native/corehost/nethost/nethost.h>
```

Reply:
Oh, I forgot about that! I'll change it.